vm/actor/src/util/unmarshallable.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use encoding::Cbor;
+use serde::de::{self, Deserializer};
+use serde::ser::{self, Serializer};
+use serde::{Deserialize, Serialize};
+
+#[derive(Default)]
+pub struct UnmarshallableCBOR {}
```

Comment:
```suggestion
pub struct UnmarshallableCBOR;
```

Doesn't need to be empty struct form, but doesn't matter. (also don't really need the default impl, because there is no data)